recipe/meta.yaml

```diff
@@ -15,8 +15,12 @@ source:
     - libmp3lame.patch
 
 build:
-  number: 1000
+  number: 1001
   skip: True  # [win]
+  run_exports:
+    # seems to be minor version compatibility
+    # https://abi-laboratory.pro/tracker/timeline/lame/
+    - {{ pin_subpackage('lame', max_pin='x.x') }}
```

Comment:
So we are ignoring [this exception]( https://abi-laboratory.pro/index.php?view=objects_report&l=lame&v1=3.97&v2=3.98.4 )? Probably reasonable as that was almost a decade ago and they have kept things a bit more stable since.

Reply:
Isn't 3.97 -> 3.98 also captured by the x.x pin?